app/frontend/src/components/ErrorFallback/ErrorFallback.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, CardActions, Typography } from "@material-ui/core";
+import Button from "components/Button";
+import PageTitle from "components/PageTitle";
+import BugReport from "features/BugReport";
+import { useCallback } from "react";
+import { NavLink, useHistory } from "react-router-dom";
+import { baseRoute } from "routes";
+import makeStyles from "utils/makeStyles";
+
+import {
+  ERROR_HEADING,
+  ERROR_INFO,
+  GO_TO_HOMEPAGE,
+  REFRESH_PAGE,
+} from "./constants";
+
+const useStyles = makeStyles((theme) => ({
+  actions: {
+    display: "flex",
+    justifyContent: "flex-end",
+    marginTop: theme.spacing(2),
+  },
+  report: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export default function ErrorFallback() {
+  const { actions, report } = useStyles();
+
+  const history = useHistory();
+
+  const handleRefresh = useCallback(() => history.go(0), [history]);
+
+  return (
+    <>
+      <PageTitle>{ERROR_HEADING}</PageTitle>
+      <Typography variant="body1">{ERROR_INFO}</Typography>
+      <Box className={report}>
+        <BugReport isResponsive={false} />
+      </Box>
+
+      <CardActions className={actions}>
```

Comment:
Card actions should only be in a card

Reply:
Is there any other component I can use for this cause? I went looking in the designs to find something similar but didn't see much except for 'NewHostRequest.tsx' which also uses CardActions. But it should be only used within a card indeed.

I'll change it here, and maybe also on NewHostRequest?

Reply:
nm I see NewHostRequest itself is being wrapped in a Card component by it's parent.
I've created an Actions component that wraps these buttons when used outside of a Card. If that's too much I can move the styling inside the ErrorFallback component.